src/commands/debug.ts

```diff
@@ -70,6 +71,12 @@ export class DebugCommands {
 			} else if (e.event === "dart.flutter.firstFrame") {
 				// Send the current value to ensure it persists for the user.
 				this.sendAllServiceSettings();
+			} else if (e.event === "dart.debugMetrics") {
+				const memory = e.body.memory;
+				const message = `${Math.ceil(memory.current / 1024 / 1024)}MB of ${Math.ceil(memory.total / 1024 / 1024)}MB`;
```

Comment:
If we're very tight on space, we could just show the current heap usage (`78MB`).

Reply:
I don't think it looks too bad like this for now (though admittedly my screen is quite wide..)

<img width="316" alt="screen shot 2018-04-03 at 08 44 55" src="https://user-images.githubusercontent.com/1078012/38236207-5d5f985a-371b-11e8-90df-1a24b9f26403.png">